src/test/php/PHPMD/TextUI/CommandTest.php

```diff
@@ -222,7 +222,7 @@ public static function dataProviderWithFilter(): array
     {
         return [
             ['--suffixes', '.class.php'],
-            ['--exclude', 'ccn_,npath_,parse_error'],
+            ['--exclude', 'ccn_,*npath_,*parse_error'],
```

Comment:
```suggestion
            ['--exclude', 'ccn_*,npath_*,*parse_error'],
```
I guess those are prefix? Not file names ending with `_`

Reply:
yeah, but the pattern new pattern only cares about prefixes :)

Reply:
Ah I see, you can force-push to revert this commit.

Reply:
Done